Model/Rate/Result.php

```diff
@@ -221,14 +271,16 @@ private function createTitle($settingPath)
      * Create price
      * Calculate price if multiple options are chosen
      *
-     * @param $alias
      * @param $settingPath
      *
      * @return float
      */
-    private function createPrice($alias, $settingPath)
+    private function getPrice($settingPath): float
```

Comment:
Je moet in php float zoveel mogelijk vermijden:
https://www.php.net/manual/en/language.types.float.php

Reply:
Er staat alleen maar dat ik floats met tig decimalen niet moet vergelijken, wat ik ook niet doe. Wat wil je dat ik hiermee doe dan? Waar ik het bij optel was al een float (/double)